### PR TITLE
Korrekturen

### PR DESCRIPTION
**Pull request by Sabine-Schoenau**

### CODE REVIEW

**Review by jmfrenzel — CHANGES_REQUESTED** [review]
Can not judge on the content of the files. Technically they look good. 

Before merging I would be great if  README was provided for nfdi4ing shared catalogs documenting 
* the content, e.g. sub directory content and file content such as catalog, conditions ect. 
* compatibility or dependency with / on  community content from the  `rdmorganiser` directory, e.g. domain options ect.

**Comment by Zack-83** [comment]
> Can not judge on the content of the files. Technically they look good.
> 
> Before merging I would be great if README was provided for nfdi4ing shared catalogs documenting
> 
>     * the content, e.g. sub directory content and file content such as catalog, conditions ect.
> 
>     * compatibility or dependency with / on  community content from the  `rdmorganiser` directory, e.g. domain options ect.

Beispiel: https://github.com/rdmorganiser/rdmo-catalog/blob/master/shared/Max-Planck-Digital-Library/Software-Management-Plan/readme.md

File: shared/nfdi4ing/NFDI4Ing_template/NFDI4Ing_Vorlage.xml [hunk 1]
```diff
@@ -8,11 +8,11 @@ Aufgrund der Darstellungsmöglichkeiten in RDMO wurde an zwei Stellen die Reihen
...
-		<order>35</order>
-		<title lang="en">NFDI4Ing_template</title>
-		<help lang="en">Catalog according to the DFG checklist of December 21, 2021</help>
-		<title lang="de">NFDI4Ing_Vorlage</title>
-		<help lang="de">Katalog entsprechend der DFG-Checkliste vom 21.12.2021</help>
+		<order>40</order>
+		<title lang="en">Generic: NFDI4Ing_template</title>
+		<help lang="en">The basis for the catalog is also the DFG checklist of December 21, 2021 with 23 questions. The template was enriched with engineer-specific content.</help>
+		<title lang="de">Generisch: NFDI4Ing_Vorlage</title>
+		<help lang="de">Als Grundlage für den Katalog dient ebenfalls die DFG-Checkliste vom 21.12.2021 mit 23 Fragen. Angereichert wurde das Template mit ingenieurspezifischen Inhalten.</help>
```
Comment (Zack-83) [inline shared/nfdi4ing/NFDI4Ing_template/NFDI4Ing_Vorlage.xml:15]:
```suggestion
		<title lang="en">NFDI4Ing_template</title>
		<help lang="en">The basis for the catalog is also the DFG checklist of December 21, 2021 with 23 questions. The template was enriched with engineer-specific content.</help>
		<title lang="de">NFDI4Ing_Vorlage</title>
		<help lang="de">Als Grundlage für den Katalog dient ebenfalls die DFG-Checkliste vom 21.12.2021 mit 23 Fragen. Angereichert wurde das Template mit ingenieurspezifischen Inhalten.</help>
```

**Review by Zack-83 — APPROVED** [review]

**Review by Zack-83 — APPROVED** [review]

File: shared/nfdi4ing/NFDI4Ing_template/NFDI4Ing_Vorlage.xml [hunk 2]
```diff
@@ -999,54 +999,22 @@ Die Bedeutung der Datenpublikation wird noch einmal unterstrichen durch die
...
-		<value_type>text</value_type>
-		<maximum/>
-		<minimum/>
-		<step/>
-		<unit/>
-		<width/>
-		<optionsets/>
-		<conditions/>
-	</question>
-	<question dc:uri="https://rdmo.nfdi4ing.de/terms/questions/NFDI4Ing_Vorlage/exchange_access/archiving/archiving_yes">
```
Comment (jmfrenzel) [inline shared/nfdi4ing/NFDI4Ing_template/NFDI4Ing_Vorlage.xml:1033]: This question has been removed.

**Review by jmfrenzel — APPROVED** [review]
Approval form my side as a Read-me file has been provided. 

I noticed that one question was removed. For me that removal looks unintended.

**Pull request merged**